src/ucs/sys/compiler_def.h

```diff
@@ -74,10 +74,10 @@
 #endif
 
 /* The i-th bit */
-#define UCS_BIT(i)               (1ul << (i))
+#define UCS_BIT(i)  (1ul << (i))
 
 /* Mask of bits 0..i-1 */
-#define UCS_MASK(i)              (UCS_BIT(i) - 1)
+#define UCS_MASK(i) ((i == 0) ? 0 : (UCS_BIT(i - 1) | (UCS_BIT(i - 1) - 1)))
```

Comment:
1. why this method it better than the current definition of UCS_MASK_SAFE?
```c
    (((_i) >= 64) ? ((uint64_t)(-1)) : (UCS_BIT(_i) - 1))
```
2. shall we delete UCS_MASK_SAFE macro?